<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>title</string>
  </property>
  <property name="windowIcon">
   <iconset resource="WorktimeManager.qrc">
    <normaloff>:/images/database.png</normaloff>:/images/database.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*[mandatoryField=&quot;true&quot;] { background-color: yellow }

 QGroupBox {
   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E0E0E0, stop: 1 #FFFFFF);
   border-radius: 5px;
   margin-top: 1ex; /* leave space at the top for the title */
   font: italic 9pt &quot;Arial&quot;;
   font-weight: bold;
   color: black;
}

QGroupBox#myGb[urgent=false] { border: 2px solid gray;	}
QGroupBox#myGb[urgent=true] { border: 2px dashed gray;	}
/*QTableView#myTv[prop=&quot;rank1&quot;] { background: red; }
QTableView#myTv[prop=&quot;rank2&quot;] { background: blue; }*/

QGroupBox::title {
   subcontrol-origin: margin; /* margin boarder padding content */
   subcontrol-position: top center; /* position at the top center */
   top: 3px;   
   padding: 0px 6px
} 
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="rightMargin">
     <number>18</number>
    </property>
    <item row="0" column="0">
     <widget class="QTextEdit" name="teInfoBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>458</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Setup">
    <property name="title">
     <string>&amp;Setup</string>
    </property>
    <widget class="QMenu" name="menuS_tyleSheets">
     <property name="title">
      <string>S&amp;tyleSheets</string>
     </property>
     <addaction name="actButGbStyleShtA"/>
     <addaction name="actButGbStyleShtInpFrm"/>
    </widget>
    <widget class="QMenu" name="menuSchriftfont">
     <property name="title">
      <string>Schriftfont</string>
     </property>
     <addaction name="actButSelFont"/>
    </widget>
    <addaction name="menuS_tyleSheets"/>
    <addaction name="menuSchriftfont"/>
    <addaction name="actSetAlterRowCol"/>
    <addaction name="actButCyclicObjInfo"/>
    <addaction name="actResizerDlg"/>
    <addaction name="actHideSqlQuery"/>
    <addaction name="actCfgInpFrmTabOrd"/>
    <addaction name="separator"/>
    <addaction name="actAutoFitTables"/>
    <addaction name="actFilterTable"/>
    <addaction name="actFilterTableWindow"/>
    <addaction name="actButClose"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actFilterForm"/>
   </widget>
   <addaction name="menu_Setup"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actButNew"/>
   <addaction name="actButOpen"/>
   <addaction name="actButSave"/>
   <addaction name="actButExport"/>
   <addaction name="actButBrowseSQL"/>
   <addaction name="actButInpForm"/>
   <addaction name="actButShowTbl"/>
   <addaction name="actButDbModMaster"/>
   <addaction name="actButClose"/>
   <addaction name="actButUnderConstr"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actButNew">
   <property name="icon">
    <iconset resource="WorktimeManager.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create a new file</string>
   </property>
  </action>
  <action name="actButOpen">
   <property name="icon">
    <iconset resource="WorktimeManager.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open an existing file </string>
   </property>
  </action>
  <action name="actButSave">
   <property name="icon">
    <iconset resource="WorktimeManager.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save the document</string>
   </property>
  </action>
  <action name="actButExport">
   <property name="icon">
    <iconset resource="WorktimeManager.qrc">
     <normaloff>:/images/export-3.png</normaloff>:/images/export-3.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export dataset to document</string>
   </property>
  </action>
  <action name="actButBrowseSQL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="WorktimeManager.qrc">
     <normaloff>:/images/database.png</normaloff>:/images/database.png</iconset>
   </property>
   <property name="text">
    <string>BrowseSQL</string>
   </property>
   <property name="toolTip">
    <string>Browser for SQL database</string>
   </property>
  </action>
  <action name="actButInpForm">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="WorktimeManager.qrc">
     <normaloff>:/images/databaseSubmit.png</normaloff>:/images/databaseSubmit.png</iconset>
   </property>
   <property name="text">
    <string>InpForm</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Öffnet die &lt;span style=&quot; font-weight:600;&quot;&gt;Eingabeform&lt;/span&gt; um neue Einträge in die in die Arbeitszeitentabelle einzutragen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C, E</string>
   </property>
  </action>
  <action name="actButShowTbl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="WorktimeManager.qrc">
     <normaloff>:/images/databaseDelegate.png</normaloff>:/images/databaseDelegate.png</iconset>
   </property>
   <property name="text">
    <string>ShowTbl</string>
   </property>
   <property name="toolTip">
    <string>Open work time database table</string>
   </property>
  </action>
  <action name="actButDbModMaster">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="WorktimeManager.qrc">
     <normaloff>:/images/databaseConf.png</normaloff>:/images/databaseConf.png</iconset>
   </property>
   <property name="text">
    <string>dbModMaster</string>
   </property>
   <property name="toolTip">
    <string>Stammdaten erweitern</string>
   </property>
  </action>
  <action name="actButClose">
   <property name="icon">
    <iconset resource="WorktimeManager.qrc">
     <normaloff>:/icoDelete</normaloff>:/icoDelete</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Alle Fenster schließen </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actButGbStyleShtA">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>gbStyleShtA</string>
   </property>
  </action>
  <action name="actButGbStyleShtInpFrm">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>gbStyleShtInpFrm</string>
   </property>
  </action>
  <action name="actButUnderConstr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="WorktimeManager.qrc">
     <normaloff>:/images/underConstruct.svg</normaloff>:/images/underConstruct.svg</iconset>
   </property>
   <property name="text">
    <string>UnderConstr</string>
   </property>
   <property name="toolTip">
    <string>Call an &quot;under construction&quot; widgets</string>
   </property>
  </action>
  <action name="actButSelFont">
   <property name="text">
    <string>Schriftfont</string>
   </property>
   <property name="toolTip">
    <string>Font / Schriftart wählen</string>
   </property>
  </action>
  <action name="actButCyclicObjInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>cyclic curso obj info</string>
   </property>
  </action>
  <action name="actResizerDlg">
   <property name="text">
    <string>Resizer DlgBox</string>
   </property>
   <property name="toolTip">
    <string>Resize MainWindow to x% of Desktop area</string>
   </property>
  </action>
  <action name="actHideSqlQuery">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide SQL query input</string>
   </property>
   <property name="toolTip">
    <string>Hide or show the input field to submit manuall SQL querys thru database driver backend</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q, Ctrl+H</string>
   </property>
  </action>
  <action name="actSetAlterRowCol">
   <property name="text">
    <string>alternating row color</string>
   </property>
   <property name="toolTip">
    <string>Farbe für alternierenden Zeilenhintergund</string>
   </property>
  </action>
  <action name="actCfgInpFrmTabOrd">
   <property name="text">
    <string>Config inpFrm tab order</string>
   </property>
  </action>
  <action name="actAutoFitTables">
   <property name="text">
    <string>Autofit Rows/Colums</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+A</string>
   </property>
  </action>
  <action name="actFilterTable">
   <property name="text">
    <string>filter table</string>
   </property>
  </action>
  <action name="actFilterTableWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>filter table window</string>
   </property>
  </action>
  <action name="actFilterForm">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter Anwenden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="WorktimeManager.qrc"/>
 </resources>
 <connections/>
</ui>
